### PR TITLE
Wait for cuda installation on Windows

### LINKED ISSUE #56654: Windows GPU testing is broken in CircleCI
See:
https://app.circleci.com/pipelines/github/pytorch/pytorch/306591/workflows/2b2a8987-831c-4704-a888-b3c7c3ebe578/jobs/12703407
And "Re-run with SSH" functionality is broken as well, making it impossible to debug what is going on:
https://support.circleci.com/hc/en-us/requests/89455

cc @ezyang @gchanan @zou3519 @bdhirsh @jbschlosser @anjali411 @seemethere @malfet @walterddr @pytorch/pytorch-dev-infra @peterjc123 @mszhanyi @skyline75489 @nbcsm

### PR DESCRIPTION
**Pull request by skyline75489**
Fixes #56654 

In `windows_cuda_install.sh`, `setup.exe` will not block the script execution. We should explicitly wait a bit for CUDA installation to finish.

### CODE REVIEW

**Comment by peterjc123** [comment]
> In `windows_cuda_install.sh`, `setup.exe` will not block the script execution. We should explicitly wait a bit for CUDA installation to finish.

I wonder how you get this conclusion. When you enable `-s` (aka `silent`) for the CUDA installer, it should be blocking. Also, in the log, it said that `import torch` fails with `Found no NVIDIA driver on your system`, but we didn't install the driver in `windows_cuda_install.sh`. I think is that fact that the GPU driver isn't properly installed (well, actually, it should do) in the host machine is most likely the cause. Next time, maybe we should check whether the GPU driver is installed or not even for CUDA 10 jobs [here](https://github.com/pytorch/pytorch/blob/master/.circleci/scripts/windows_cuda_install.sh#L28).

File: .jenkins/pytorch/win-test.sh [hunk 1]
```diff
@@ -57,9 +57,9 @@ fi
 
 run_tests() {
     # Run nvidia-smi if available
-    for path in  /c/Program Files/NVIDIA Corporation/NVSMI/nvidia-smi.exe /c/Windows/System32/nvidia-smi.exe; do
-        if [ -x $path ]; then
-            $path;
+    for path in  "/c/Program Files/NVIDIA Corporation/NVSMI/nvidia-smi.exe" "/c/Windows/System32/nvidia-smi.exe"; do
```
Comment (peterjc123) [inline .jenkins/pytorch/win-test.sh:60]: Thanks for fixing this one. Maybe you should do it in a separate PR.

File: .circleci/scripts/windows_cuda_install.sh [hunk 2]
```diff
@@ -40,6 +40,8 @@ set +e
 
 ./setup.exe -s ${cuda_install_packages} -loglevel:6 -log:"$(pwd -W)/cuda_install_logs"
 
+sleep 180
```
Comment (malfet) [inline .circleci/scripts/windows_cuda_install.sh:43]: Why is this necessary? And how do we know that 180 sec is sufficient?

File: .jenkins/pytorch/win-test.sh (hunk 1 above)
Comment (skyline75489) [reply .jenkins/pytorch/win-test.sh:60]: will do
